### ISSUE TITLE
Training & validation accuracy is 0

### ISSUE THREAD

**Issue by devvikas** [makatx/YOLO_ResNet#7]
Very soon after we start the training, the accuracy goes to 0. Why is that ?

Epoch 1/15
115/115 [==============================] - 696s 6s/step - loss: 12.9728 - **acc: 0.0094 - val_loss: 11.6941 - val_acc: 0.0000e+00**
Epoch 00001: val_loss improved from inf to 11.69414, saving model to models/run1_0.01_weights.01-11.69.hdf5
Epoch 2/15
 91/115 [======================>.......] - ETA: 2:15 - loss: 6.9046 - **acc: 0.0000e+00**
